### PR TITLE
Updated copy for No results view full screen displayed on connection issue

### PR DESCRIPTION
**Pull request by yaelirub**
Fixes: #11030
We want to show the same copy when the full screen (no results view) is displayed in case of connectivity issue.

To test:
1. Run the app.
2. Go offline
3. Go over any of the full pages (stats, activity, site pages...) and see that the copy for the title and subtitle is consistent .
4. Check that the full screen copy hasn't changed (as it was before this PR) in case the screen appears due to other reason for example:
4.1. Create a new site
4.2. Go to Media.
4.3. See that the illustration and the copy are media specific.
4.4. Go to site pages.
4.5. See that the illustration and the copy are site pages specific.
4.6. Go to Drafts tab.
4.7. See that the copy is draft pages specific.
4.8. Go to blog posts
4.9. Delete the About post.
4.10. See that the illustration and the copy are Blog posts specific.
etc'

* This PR does not include the updated illustration
  Update release notes:

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift [hunk 1]
```diff
@@ -48,6 +69,12 @@ import WordPressShared
     private var titleLabelMaxWidthConstraint: NSLayoutConstraint?
     private var titleLabelTopConstraint: NSLayoutConstraint?
 
+    //For No results on connection issue
+    private var reachability = Reachability.forInternetConnection()
```
Comment (shiki) [inline WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift]: It looks like we have no intention of mutating `reachability` so we can probably use `private let` for this.

File: WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift [hunk 2]
```diff
@@ -516,4 +544,9 @@ private extension NoResultsViewController {
     func stopAnimatingIfNeeded() {
         stopAnimatingViewIfNeeded(accessorySubview)
     }
+
+    struct NoConnectionMessages {
```
Comment (shiki) [inline WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift]: I like how we group constants this way 👍

**Comment by yaelirub** [comment]
@shiki  updated

File: WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift [hunk 3]
```diff
@@ -15,6 +15,7 @@ import Reachability
 ///     - hideImage is set to true.
 /// The action button is shown by default, but will be hidden if button title is not provided.
 /// The subtitle is optional and will only show if provided.
+/// If this view is presented as a result of connectivity issue we will override the title, subtitle and image to default values defined in the NoConnection struct
```
Comment (shiki) [inline WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift]: 👍

File: WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift [hunk 4]
```diff
@@ -70,7 +51,7 @@ import Reachability
     private var titleLabelTopConstraint: NSLayoutConstraint?
 
     //For No results on connection issue
-    private var reachability = Reachability.forInternetConnection()
+    private let reachability = Reachability.forInternetConnection()
     private var isConnectionAvailable: Bool? {
```
Comment (shiki) [inline WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift]: I guess we can get rid of this now 🙂

File: WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift [hunk 5]
```diff
@@ -545,8 +521,9 @@ private extension NoResultsViewController {
         stopAnimatingViewIfNeeded(accessorySubview)
     }
 
-    struct NoConnectionMessages {
+    struct NoConnection {
         static let title: String = NSLocalizedString("Unable to load this page right now.", comment: "Title for No results full page screen displayed when there is no connection")
         static let subTitle: String = NSLocalizedString("Check your network connection and try again.", comment: "Subtitle for No results full page screen displayed when there is no connection")
+        static let imageName = "cloud"
```
Comment (shiki) [inline WordPress/Classes/ViewRelated/Blog/NoResultsViewController.swift]: 💯

**Pull request merged**